tests/itest_specs/deposit_liquidity.cairo

```diff
@@ -1,5 +1,581 @@
+                "./contracts/option_token.cairo",
+                [1234, 14, 18, 0, 0, context.admin_address, context.amm_addr, context.myusd_address, context.myeth_address, 1, ids.strike_price, expiry_longer, 1]
+            ).contract_address
+
+            ids.opt_long_call_addr_2 = deploy_contract(
+                "./contracts/option_token.cairo",
+                [1234, 14, 18, 0, 0, context.admin_address, context.amm_addr, context.myusd_address, context.myeth_address, 0, ids.strike_price, expiry_longer, 0]
+            ).contract_address
+
+            ids.opt_long_put_addr_2 = deploy_contract(
+                "./contracts/option_token.cairo",
+                [1234, 14, 18, 0, 0, context.admin_address, context.amm_addr, context.myusd_address, context.myeth_address, 1, ids.strike_price, expiry_longer, 0]
+            ).contract_address
+        
+        %}
+        let hundred_m64x61 = Math64x61.fromFelt(100);
+        // Add short long option with longer maturity
+        ILiquidityPool.add_option(
+            contract_address=amm_addr,
+            option_side=1,
+            maturity=expiry_longer,
+            strike_price=strike_price,
+            quote_token_address=myusd_addr,
+            base_token_address=myeth_addr,
+            option_type=0,
+            lptoken_address=lpt_call_addr,
+            option_token_address_=opt_short_call_addr_2,
+            initial_volatility=hundred_m64x61
+        );
+        // Add short put option with longer maturity
+        ILiquidityPool.add_option(
+            contract_address=amm_addr,
+            option_side=1,
+            maturity=expiry_longer,
+            strike_price=strike_price,
+            quote_token_address=myusd_addr,
+            base_token_address=myeth_addr,
+            option_type=1,
+            lptoken_address=lpt_put_addr,
+            option_token_address_=opt_short_put_addr_2,
+            initial_volatility=hundred_m64x61
+        );
+        // Add options for other side as well
+        ILiquidityPool.add_option(
+            contract_address=amm_addr,
+            option_side=0,
+            maturity=expiry_longer,
+            strike_price=strike_price,
+            quote_token_address=myusd_addr,
+            base_token_address=myeth_addr,
+            option_type=0,
+            lptoken_address=lpt_call_addr,
+            option_token_address_=opt_long_call_addr_2,
+            initial_volatility=hundred_m64x61
+        );
+        // Add short put option with longer maturity
+        ILiquidityPool.add_option(
+            contract_address=amm_addr,
+            option_side=0,
+            maturity=expiry_longer,
+            strike_price=strike_price,
+            quote_token_address=myusd_addr,
+            base_token_address=myeth_addr,
+            option_type=1,
+            lptoken_address=lpt_put_addr,
+            option_token_address_=opt_long_put_addr_2,
+            initial_volatility=hundred_m64x61
+        );
```

Comment:
btw, there was no need to add options... you could have just used the existing ones